R-packages/evalcast/R/plot_calibration.R

```diff
@@ -2,8 +2,7 @@
 #'
 #' @param scorecard Single score card.
 #' @param type One of "wedgeplot" or "traditional".
-#' @param alpha Parameter defining nominal interval coverage.
-#' @param legend_position Legend position, the default being "bottom".
+#' @param legend.position Legend position, the default being "bottom".
```

Comment:
If we're leaving `alpha` in as an argument for now, perhaps we can add the following?

```
@param alpha Old parameter that will be removed soon.
```

Reply:
We should do this so that R CMD check doesn't complain.

Reply:
Done.  (Here and in `plot_width()`.)